views/profile.pug

```diff
@@ -0,0 +1,12 @@
+extends layout.pug
+
+block content
+  .card
+    .card-body
+      h1.card-title= theUser.displayName
+      if (theUser)
+        .card-subtitle.mb-2.text-muted= theUser.roleName
+        br
```

Comment:
You should almost never have to use BR. You should have .card-text on the `p` below